matchzoo/models/drmm.py

```diff
@@ -0,0 +1,107 @@
+"""An implementation of DRMM Model."""
+import typing
+
+import keras
+import keras.backend as K
+from keras.activations import softmax
+
+from matchzoo import engine
+
+
+class DRMM(engine.BaseModel):
```

Comment:
Use the same naming convension, e.g. `DSSMModel `, `CDSSMModel`.
```suggestion
class DRMMModel(engine.BaseModel):
```

Reply:
We will rename all models and remove the `Model`. In this way, all models can be called such as `matchzoo.models.DRMM`.